x/cli/argument_command.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"io"
+)
+
+type ArgumentCommand struct {
```

Comment:
```suggestion
type Argument struct {
```

or

```suggestion
type CommandArgument struct {
```

WDYT ?

Reply:
`XyzCommand` is a pattern here to tell you what is a command, checkout the godoc it makes it very clear https://godoc.org/github.com/upfluence/cfg/x/cli (SubCommand, StaticCommand) same goes for the private ones